File: Go/01.Go-Basic-February-2022/05.conditionalStatementAdvanced-Lab/03.animalType.go
package main

import "fmt"

func main() {
	var animal string
	fmt.Scanln(&animal)

	switch animal {
	case "dog":
		fmt.Print("mammal")
	case "crocodile", "tortoise", "snake":
		fmt.Print("reptile")
	default:
		fmt.Print("unknown")
	}
}

File: JavaScript/01.JavaScript-Basic-February-2021/05.conditionalStatementsAdvanced-Lab/03.animalType.js
function animalType(input) {
    let animal = input[0];

    if (animal == "dog") {
        console.log("mammal");

    } else if (animal == "crocodile" || animal == "tortoise" || animal == "snake") {
        console.log("reptile");
    } else {
        console.log("unknown");
    }
}

animalType(["dog"]);
animalType(["snake"]);
animalType(["cat"]);